test/framework/common_constants.cpp

```diff
@@ -32,8 +32,8 @@ namespace common_constants {
   // ids
   const std::string kAdminId = kAdminName + "@" + kDomain;
   const std::string kUserId = kUser + "@" + kDomain;
-  const std::string kSameDomainUserId = kAnotherUser + "@" + kDomain;
-  const std::string kAnotherDomainUserId = kAnotherUser + "@" + kSecondDomain;
+  const std::string kSameDomainUserId = kSameDomainUser + "@" + kDomain;
+  const std::string kSecondDomainUserId = kSameDomainUser + "@" + kSecondDomain;
```

Comment:
Sorry, but the new naming is very misleading at that place. "second domain user" is "same domain user" at "second domain".

For me, it looks like a contradiction: "second domain" = "same domain" + ...

If you can introduce a better naming scheme, then please do so.

Reply:
thanks, it was a bad naming